<launch>
  <!-- No namespace here as we will share this description. 
       Access with slash at the beginning -->
  <param name="robot_description"
    command="$(find xacro)/xacro.py $(find turtlebot3_description)/urdf/turtlebot3_burger.urdf.xacro" />

  <!-- BEGIN ROBOT 1-->
  <group ns="robot1">
    <param name="tf_prefix" value="robot1_tf" />
    <include file="$(find robot_chase)/launch/one_robot.launch" >
      <arg name="init_pose" value="-x 6 -y 4 -z 0" />
      <arg name="robot_name"  value="robot1" />
    </include>
  </group>

  <!-- BEGIN ROBOT 2-->
  <group ns="robot2">
    <param name="tf_prefix" value="robot2_tf" />
    <include file="$(find robot_chase)/launch/one_robot.launch" >
      <arg name="init_pose" value="-x -1.8 -y 5.7 -z 0" />
      <arg name="robot_name"  value="robot2" />
    </include>
  </group>

  <group ns="robot3">
    <param name="tf_prefix" value="robot3_tf" />
    <include file="$(find robot_chase)/launch/one_robot.launch" >
      <arg name="init_pose" value="-x -3.8 -y -4.2 -z 0" />
      <arg name="robot_name"  value="robot3" />
    </include>
  </group>

  <group ns="robot4">
    <param name="tf_prefix" value="robot4_tf" />
    <include file="$(find robot_chase)/launch/one_robot.launch" >
      <arg name="init_pose" value="-x 6 -y -3 -z 0" />
      <arg name="robot_name"  value="robot4" />
    </include>
  </group>
</launch>